core/local/index.js

```diff
@@ -146,14 +153,25 @@ module.exports = class Local /*:: implements Side */ {
   inodeSetter (doc /*: Metadata */) {
```

Comment:
We could try to come up with a different name for this function as it doesn't set the inode only anymore.
Something in the likes of `identifiersSetter`?

Reply:
`identifier` makes me think to the `id`, that are built from the `path`, not from the `inode` and `fileId`. Do you see another name that could fit?

Reply:
I moved everything in the new `stater` module